### DISCUSSION TITLE
Kaboom replit misfeature? Subdirs of code/ not shown in Kaboom Code menu

### DISCUSSION THREAD

**Discussion by dbarnett** [replit/kaboom#618] [Q&A]
When you fork the official Kaboom template (https://replit.com/@replit/Kaboom), there's a "Kaboom" section in the menu that lists the files under code/ nicely next to sprites etc, but it doesn't find any files in a subdirectory like code/scenes/. Could those be surfaced too?

**Comment by lajbel** [comment]
Yes, this is something that is not yet implemented in the replit editor. I don't know if it will be implemented, but it could be.

What you can do is to put a prefix in each code file, like scene_game, obj_player, and so on!

**Comment by dbarnett** [comment]
Yeah, that's exactly what I ended up doing. I was hoping I could find code for the replit template somewhere and figure out how to fix it properly, but couldn't find it.
